SR2023/2023-02-13-virtual-competition-code-submission.md

```diff
@@ -0,0 +1,35 @@
+---
+to: Student Robotics 2023
+subject: Submit your robot for the SR2023 Virtual Competition
+---
+
+Hi
+
+You can now submit your robot code through our website for the [virtual competition](https://studentrobotics.org/events/sr2023/virtual-competition/).
+
+[Submit your robot code](https://studentrobotics.org/code-submitter/).
+
+You have until 8pm (UK) Friday 24th February to submit your team's latest code. You can upload early, but at that time we'll capture all the uploads. If you fail to upload any code, you will be marked as absent from all the matches and score 0 points. This will not affect your ability to compete in the [final competition](https://studentrobotics.org/events/sr2023/competition/).
+
+Simply log in with your TLA (You can find this from your channel name in Discord, removing `team-`, in upper-case) and the password for your team's Discord channel we sent at the start of the competition year. If you need a refresher, please let us know in your team channel.
+
+Once in, you can upload a `.zip` of your code. This zip must contain a `robot.py` file, and any other supporting code or data files your robot may need:
+
+```
+  robot.zip
+  ├── secret_magic.py
+  └── robot.py
+```
+
+As there is just 1 chance to submit, we have a few suggestions for things to keep an eye out for:
```

Comment:
This makes it sound like they can only perform the upload once, which isn't the case. Is there a framing which avoids that?